discord/gateway.py

```diff
@@ -827,7 +827,7 @@ def __init__(
         self.loop: asyncio.AbstractEventLoop = loop
         self._keep_alive: Optional[VoiceKeepAliveHandler] = None
         self._close_code: Optional[int] = None
-        self.secret_key: Optional[str] = None
+        self.secret_key: Optional[List[int]] = None
```

Comment:
This is most likely `bytes` no?

Reply:
In the one place it's actually used (making the SecretBox) it gets cast to bytes on use.  I could refactor everything to change it to bytes on arrival but it was just this hint that was inconsistent with `List[int]`.